lib/environment.rb

```diff
@@ -439,7 +439,7 @@ def service_project
       unless @service_project
         # if the cluster set the default scheduler, set the project running debug pod node-selector=''
         # to overwrite the default scheduler, or the pod can not be run successfully
-        project_name = "proj-" + EXECUTOR_NAME.downcase
+        project_name = "pj-" + EXECUTOR_NAME.downcase
```

Comment:
What about `prj`, `pj` looks like a name short for `prow job` ;)

Reply:
Updated